plugins/woocommerce/includes/admin/meta-boxes/views/html-product-data-inventory.php

```diff
@@ -62,7 +62,7 @@
 				)
 			);
 
-			echo '<input type="hidden" name="_original_stock" value="' . esc_attr( wc_stock_amount( $product_object->get_stock_quantity( 'edit' ) ) ) . '" />';
+			echo '<input type="hidden" name="_original_stock" value="' . esc_attr( wc_stock_amount( $product_object->get_stock_quantity( 'edit' ) ?? 1 ) ) . '" />';
```

Comment:
Again, this will set the original stock value to `1` even when it really is `0`. And results in an error message when saving, since we are altering the "original" value. I think we can just revert this change and things will work properly (I tried it locally).

<img width="1106" alt="Screenshot 2023-03-21 at 12 41 54" src="https://user-images.githubusercontent.com/2098816/226684092-974ed3b6-9a8b-457f-aad4-4d13819ae286.png">

Reply:
Thank you for the review. I'm still confused about how I managed to test this PR before pushing it. I guess I used the wrong branch 🤦🏻‍♂️

The problem was that I was setting the `_original_stock` as `1` but during the check [when saving](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/admin/meta-boxes/class-wc-meta-box-product-data.php#L344) it was getting a `0`, and this is why we saw the error.
I fixed that in commit [02ca669](https://github.com/woocommerce/woocommerce/pull/37304/commits/02ca66943bda161cf4c042435aff1b6e4e36fadd). The `_original_stock` shouldn't change.